### PR TITLE
Blog app - creating data model

### PR DESCRIPTION
**Pull request by abdullah-asghar-khan**
- Created a new app.

- Initialized this project with Git.

- Make sure that your project has a Postgres database set up.

- Build your project schema.

- Create and run the necessary migration files.

### CODE REVIEW

File: db/migrate/20220913150720_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
```
Comment (tufoinnkuo10) [inline db/migrate/20220913150720_create_users.rb:8]: - [ ] Great job.a user should control the post with a postcounter. you are missing the post counter 
eg `t.integer :posts_counter`

File: db/migrate/20220913150929_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
```
Comment (tufoinnkuo10) [inline db/migrate/20220913150929_create_posts.rb]: - [ ] coments and likes will be created on a post. you are missing the counters
eg
```ruby
    t.integer :comments_counter
    t.integer :likes_counter
```

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Abdullah2213565 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Abdullah2213565,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**